code/modules/telesci/gps.dm

```diff
@@ -79,3 +79,8 @@ var/list/GPS_list = list()
 	icon_state = "gps-m"
 	gpstag = "MINE0"
 	desc = "A positioning system helpful for rescuing trapped or injured miners, keeping one on you at all times while mining might just save your life."
+
+/obj/item/device/gps/internal
+	icon_state = null
+	gpstag = "Eerie Signal"
+	desc = "Report to a coder immediately."
```

Comment:
this sort of message isn't dumb tbh

i find it funny to get clever snarky remarks from coders if something breaks